lib/rb_shift/project.rb

```diff
@@ -60,54 +60,64 @@ def routes(update = false)
     end
 
     def create_secret(name, kind, **opts)
+      log.info "Creating secret #{kind} #{name}"
       execute "create secret #{kind} #{name}", **opts
       secrets true if @_secrets
     end
 
     def add_role_to_user(role, username)
+      log.info "Adding role to user #{role} #{username}"
       execute "policy add-role-to-user #{role} #{username}"
       role_bindings true if @_role_bindings
     end
 
     def add_role_to_group(role, groupname)
+      log.info "Adding role to group #{role} #{groupname}"
```

Comment:
The word order seems a bit weird.. what about `"Adding role #{role} to group #{groupname}"`

Reply:
Also put there name the project.